lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -107,10 +108,10 @@ TxMeta
     txMetaDirection         W.Direction         sql=direction
     txMetaSlot              SlotNo              sql=slot
     txMetaBlockHeight       Word32              sql=block_height
-    txMetaAmount            Natural             sql=amount
+    txMetaAmount            OverflowNatural     sql=amount
     txMetaData              W.TxMetadata Maybe  sql=data
     txMetaSlotExpires       SlotNo Maybe        sql=slot_expires
-    txMetaFee               Natural Maybe       sql=fee
+    txMetaFee               OverflowNatural Maybe sql=fee
```

Comment:
`Natural` isn't supported anymore and the `PersistField Natural` instance is an error instance :roll_eyes:, so we can't even overwrite it.